creator-node/src/diskManager.js

```diff
@@ -187,6 +195,145 @@ class DiskManager {
 
     return ret
   }
+
+  static async deleteFileOrDir(pathToFileOrDir) {
```

Comment:
worth adding test(s)? can do separately

Reply:
yes I definitely think so. I started but didn't get too far - made a note to add soon

Reply:
coo